src/main/java/task/Event.java

```diff
@@ -0,0 +1,51 @@
+package task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This class represents an event.
```

Comment:
Maybe can provide a little more detail in the comment header for the class to indicate how Todo, Deadline and Event tasks differ? Such as mentioning how Deadline has a due date and Event has from and to dates

Reply:
Thank you for your feedback! What you say makes sense and I will take it into consideration and edit it to be clearer :)